frontend/app/tests/e2e/specs/accounts-balances.spec.ts

```diff
@@ -34,65 +44,154 @@ describe('Accounts', () => {
     app.logout();
   });
 
-  function goToManualBalances() {
-    cy.get('.accounts-balances__manual-balances').click({
-      force: true
+  describe('blockchain balances', () => {
+    let blockchainBalances: FixtureBlockchainBalance[];
+    before(() => {
+      cy.fixture('blockchain-balances').then(balances => {
+        blockchainBalances = balances.map((balance: { blockchain: string }) => {
+          const address = {
+            [Blockchain.ETH]: Cypress.env('ETH_ADDRESS'),
+            [Blockchain.BTC]: Cypress.env('BTC_ADDRESS')
+          }[balance.blockchain];
+
+          return { ...balance, address };
+        });
+      });
+      cy.get('.accounts-balances__blockchain-balances').should('be.visible');
```

Comment:
This is not really a blocker at the moment but in general, it would be good if we start moving more from classes to `data-cy`.

Reply:
Yeah, I think it would be better too. I think we need to create agreement for the naming convention of the `data-cy` (Since the classname is using `BEM`, maybe we should use `BEM` too for the `data-cy`).